scripts/error_codes.py

```diff
@@ -246,7 +246,6 @@ def examine_id_coverage(top_dir, source_id_to_file_names, new_ids_only=False):
         "4802",
         "4902",
         "5272",
-        "5622",
```

Comment:
I hope it's OK to remove the error code here if it is not present in the code anymore.

Reply:
Of course. Not removing it would actually be a bug (it could silence a new error introduced later).